src/SwedbankPay.Sdk/RelationConstants.cs

```diff
@@ -16,38 +16,4 @@ public static class ConsumerResourceOperations
         public const string ViewConsumerIdentification = "view-consumer-identification";
         public const string RedirectConsumerIdentification = "redirect-consumer-identification";
     }
-
```

Comment:
`RelationConstants.cs` should be renamed to `ConsumerResourceOperations.cs` and moved to a more appropriate place.

Reply:
Done